tests/study_tests/test_study.py

```diff
@@ -300,7 +300,7 @@ def test_study_set_and_get_system_attrs(storage_mode: str) -> None:
     with StorageSupplier(storage_mode) as storage:
         study = create_study(storage=storage)
 
-        study.set_system_attr("system_message", "test")
+        study._set_system_attr("system_message", "test")
```

Comment:
It might be a bit controversial but I'd say we can just remove test scenarios that checks the behavior of `set_system_attr`. What do you think?

Reply:
Will we continue to use `set_system_attr` internally for the future? If so, I think it's better to test.

Reply:
No, we will internally use `storage.set_trial_system_attr` instead of `trial.se_system_attr`. However, as you said, it might be better to leave tests for `trial.set_system_attr`.

@gen740 (cc: @HideakiImamura ) What do you think?

Reply:
I heard that deprecated function tests are also deprecated customarily, so I deleted them.